src/FakeItEasy.Analyzer/Resources.resx

```diff
@@ -129,4 +141,4 @@
     <value>Unused call specification</value>
     <comment>The title of the diagnostic.</comment>
   </data>
-</root>
+</root>
```

Comment:
I'm a little sad to see all the distracting whitespace changes here.
For my curiosity, did you edit the file using the visual editor in the IDE? I suspect that's what introduced the changes.